src/components/FormikMultiSelect.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const MenuProps = {
+  getContentAnchorEl: null, //TODO: remove this when material-ui gets updated to version 5
+  anchorOrigin: {
+    vertical: 'bottom',
+    horizontal: 'left',
+  },
+  PaperProps: {
+    style: {
+      maxHeight: 250,
+      width: 250,
+    },
+  },
+};
+
+function FormikMultiSelect(props) {
```

Comment:
Similar to above a placeholder explaining what no selection means would be nice. E.g. "No ethnicity requirements", etc.